std/conv.d

```diff
@@ -5594,3 +5576,236 @@ unittest
     assert(hexString!"30\r\n\t\f\v31 32 33 32 31 30" == "0123210");
     assert(hexString!"ab cd" == hexString!"ABCD");
 }
+
+
+/**
+ * Convert integer to a range of characters.
+ * Intended to be lightweight and fast.
+ *
+ * Params:
+ *      Radix = 2, 8, 10, 16
+ *      Char = character type for output
+ *      letterCase = lower for deadbeef, upper for DEADBEEF
+ *      value = integer to convert. Can be uint or ulong. If Radix is 10, can also be
+ *              int or long.
+ * Returns:
+ *      Random access range with slicing and everything
+ */
+
+auto toChars(ubyte Radix = 10, Char = char, LetterCase letterCase = LetterCase.lower, T)(T value)
+    pure nothrow @nogc @safe
+    if ((Radix == 2 || Radix == 8 || Radix == 10 || Radix == 16) &&
+        (is(Unqual!T == uint) || is(Unqual!T == ulong) ||
+         Radix == 10 && (is(Unqual!T == int) || is(Unqual!T == long))))
+{
+    alias UT = Unqual!T;
+
+    static if (Radix == 10)
+    {
+        /* uint.max  is 42_9496_7295
+         *  int.max  is 21_4748_3647
+         * ulong.max is 1844_6744_0737_0955_1615
+         *  long.max is  922_3372_0368_5477_5807
+         */
+        struct Result
+        {
+            this(UT value)
+            {
+                bool neg = false;
+                if (value < 0)
+                {
+                    value = -value;
+                    neg = true;
+                }
+                size_t i = buf.length;
+                do
+                {
+                    buf[--i] = cast(ubyte)('0' + cast(Unsigned!UT)value % 10);
```

Comment:
Ehm .. ok scratch the above, but might be good idea to cast it once before the loop?